release-notes-1.2.0.md

```diff
@@ -0,0 +1,766 @@
+
+# Highlights
+
+- Redesigned XML, Stream and Query support
+- Based on a stable language specification: 2020R1
+- Improved Compilation time.
+- Includes a number of important bug fixes
+- Added cookies and trailer support in HTTP.
+- Revamped Cache module.
+- Redesigned oneof field and map field support in gRPC and Added client retry support for gRPC blocking unary calls.
+- Added custom serializer/deserializer and Avro support in the Kafka connector. 
+- Deprecated previous JDBC connector and introduced new Database connectors which are now in a preview state.
+- Introduced new client connectors (email, slack and azure time-series).
+- Changed organization names of all supported connectors.
+- Support to mock functions when writing tests
+- Test and code coverage report
+- Java binding generator tool
+
+# What's new in JBallerina 1.2.0?
+
+## Language
+
+The language implementation is based on the stable language specification version 2020R1. This implementation introduces a new set of features aimed at making integration further easier for the user. In addition to that, the XML and Lock implementations are out in their preview/experimental status.
+
+In addition to new features, this release also includes critical bug fixes and fixes done to align the implementation with the language specification. Some of these changes are backward incompatible. For a complete list of changes done since 1.1.0, see the `Language changes since JBallerina 1.1.0` section.
+
+### New XML design
+
+This release introduces a revamped XML support along with XPath-like query syntax allowing easy and safe manipulation of XML data. 
+
+#### XML navigation expression
+
+XML step expressions allow to query the children of an XML element or children of members of an XML sequence.
+
+```ballerina
+xml x = xml `<root>
+<person><name><fname>Kamal</fname><lname>Gunae</lname></name></person>
+<person><name><fname>Nimal</fname><lname>Jayee</lname></name></person>
+<person><name><fname>Sunil</fname><lname>Kumarae</lname></name></person>
+</root>`;
+
+// Select all children items of `x`.
+xml allChildren = x/*;
+
+// Select all children items, which are XML elements.
+xml allElementChildren = x/<*>;
+
+// Match first-level children with element name `person`.
+// Then, match the second level with element name `name`
+// and the third level with element name `fname`.
+xml fnameItems = x/<person>/<name>/<fname>;
+
+// Select all descendants, which match element name `fname`.
+xml fnameDescendents = x/**/<fname>;
+```
+
+XML filter expression allows filtering an XML sequence by an element name.
+
+```ballerina
+xml x = xml `<root>
+<rectangle length="5" width="10"/>
+<rectangle length="5" width="5"/>
+<circle radius="2"/>
+<square side="6"/>
+</root>`;
```

Comment:
Same comment as above regarding formatting

Reply:
@pubudu - Can you fix the formatting here itself so that I can commit directly?